05_syntax_lab/01.py

```diff
@@ -1,5 +1,11 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+#"""
+#Write a program that reads 10 numbers from
+#the user and prints the largest one
+#"""
+print "Enter 10 numbers:"
+max = 0
+for i in range(0,10):
```

Comment:
1. Consider the case if all 10 numbers a user enters are negatives. What will the program print? How can you fix it?

2. Prefer using `range(10)` over `range(0,10)` - it's shorter and cleaner